app/templates/foundry.html

```diff
@@ -34,10 +33,10 @@ <h1 class="page-title">ADI Foundry</h1>
                     <h4>Apply</h4>
                     <span class="bar gray"></span>
                 </div>
-                <h3>Applications are due by October 2nd, so apply soon!</h3>
-                <div class="button-group">
+                <h3>Applications have not opened yet. Sign up for the ADI newsletter to be notified when applications open!</h3>
```

Comment:
How about linking to the signup form: [Sign up for the ADI newsletter](http://eepurl.com/jaugD) to be notified...